saleor/dashboard/templatetags/chips.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import unicode_literals
+
+from django.template.defaultfilters import yesno
+from django.utils.translation import pgettext_lazy
+
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
+
+
+CHIPS_PATTERN = '%s: %s'
+
+
+def handle_default(field, request_get):
+    """Build a list of chips using raw field's value."""
+    return [{
+        'content': CHIPS_PATTERN % (field.label, field.value()),
+        'link': get_cancel_url(request_get, field.name)}]
+
+
+def handle_single_choice(field, request_get):
+    """Build a list of chips for ChoiceField field."""
+    for choice_value, choice_label in field.field.choices:
+        if choice_value == field.value():
+            item = {
+                'content': CHIPS_PATTERN % (field.label, choice_label),
+                'link': get_cancel_url(request_get, field.name)}
+            return [item]
+    return []
+
+
+def handle_multiple_choice(field, request_get):
+    """Build a list of chips for MultipleChoiceField field."""
+    items = []
+    for value in field.value():
+        for choice_value, choice_label in field.field.choices:
+            if choice_value == value:
+                items.append({
+                    'content': CHIPS_PATTERN % (field.label, choice_label),
+                    'link': get_cancel_url(request_get, field.name, value)})
+    return items
+
+
+def handle_single_model_choice(field, request_get):
+    """Build a list of chips for ModelChoiceField field."""
+    for obj in field.field.queryset:
+        if str(obj.pk) == str(field.value()):
+            return [{
+                'content': CHIPS_PATTERN % (field.label, str(obj)),
+                'link': get_cancel_url(request_get, field.name)}]
+    return []
+
+
+def handle_multiple_model_choice(field, request_get):
+    """Build a list of chips for ModelMultipleChoiceField field."""
+    items = []
+    for pk in field.value():
+        # iterate over field's queryset to match the selected object
+        for obj in field.field.queryset:
+            if str(obj.pk) == str(pk):
+                items.append({
+                    'content': CHIPS_PATTERN % (field.label, str(obj)),
+                    'link': get_cancel_url(request_get, field.name, pk)})
+    return items
+
+
+def handle_nullboolean(field, request_get):
+    """Build a list of chips for NullBooleanField field."""
+    value = yesno(
+        field.value(),
+        pgettext_lazy('Possible values of boolean filter', 'yes,no,all'))
```

Comment:
Please don't do this. If that's the only way to pass data do `yesno` then please don't use `yesno` as well.